protocols/kad/src/query/peers/closest/disjoint.rs

```diff
@@ -536,8 +539,8 @@ mod tests {
     struct Target(KeyBytes);
 
     impl Arbitrary for Target {
-        fn arbitrary<G: Gen>(g: &mut G) -> Self {
-            Target(Key::from(random_peers(1, g).pop().unwrap()).into())
+        fn arbitrary(_g: &mut Gen) -> Self {
+            Target(Key::from(random_peers(1, &mut rand::thread_rng()).pop().unwrap()).into())
```

Comment:
Given that we always pass the same randomness now, we can probably get rid of this argument?

Reply:
Well, the question is: do we need to get the randomness from Gen or from Rng? If from Rng, then most of that `gen_range` changes can be reverted.

Reply:
I think for `random_peers`, it is completely fine to get it from `thread_rng`.

Reply:
Done